packages/lmi/src/lmi/embeddings.py

```diff
@@ -88,6 +88,19 @@ class LiteLLMEmbeddingModel(EmbeddingModel):
             " Router is not used here."
         ),
     )
+    _router: litellm.Router | None = None
+
+    @property
+    def router(self) -> litellm.Router:
+        if self._router is None:
+            router_kwargs: dict = self.config.get("router_kwargs", {})
+            if self.config.get("pass_through_router"):
+                self._router = PassThroughRouter(**router_kwargs)
```

Comment:
I think you need to import this -- can you get `pre-commit` and `lint` CI to pass?

Reply:
Hopefully the last commit did the job.

Reply:
I just pushed a commit to fix `isort`